src/classes/UTIL_CustomSettingsFacade.cls

```diff
@@ -89,10 +89,10 @@ public without sharing class UTIL_CustomSettingsFacade {
      * Private methods that will configure the default org-level custom settings.
      **/
     private static void configSettings(Hierarchy_Settings__c hs) {
```

Comment:
@xinnanlisf This falls outside the install context. I apologize for the lack of clarity as the intent was to limit the new potential exception to a fresh install only.

My suggestion is to go to the entry point in the `STG_InstallScript.onInstall(InstallContext)` where it calls `UTIL_CustomSettingsFacade.getSettings()`, then build out parallel functionality for that code path in a separate method, ultimately calling `UTIL_CustomSettingsFacade.configSettings(Hierarchy_Settings__c)` and then setting `Account_Processor__c` and `Administrative_Account_Record_Type__c` to `UTIL_Describe.getDefaultAdminAccRecTypeId()` before the upsert.

We can leave `UTIL_Describe.getAdminAccRecTypeID()` in `UTIL_CustomSettingsFacade.configSettings(Hierarchy_Settings__c)` to prevent any unexpected behavior.